astroquery/mpc/core.py

```diff
@@ -877,29 +879,30 @@ def get_observations_async(self, targetid, *,
         (#): Parameters ``Note1`` and ``Note2`` are defined `here
         <https://minorplanetcenter.net/iau/info/OpticalObs.html>`_.
 
+        (!): Definitions of catalog codes can be found `here
+        <https://www.minorplanetcenter.net/iau/info/CatalogueCodes.html>`_.
+
```

Comment:
I see that this just copies what we already do in the previous line, but making `here` a reference breaks the sphinx build, so I would suggest to make it a link only but not a reference. 

Overall it's recommended to avoid too generic nouns to be linked, e.g. `here`, `click here`, etc, but arguably we're not exactly consistent with this.
e.g. ref https://heyoka.medium.com/dont-use-click-here-f32f445d1021

Reply:
Hi, I've turned both into plain links but the build still fails. I'm not sure what the problem could be?